types/selectize/index.d.ts

```diff
@@ -436,18 +444,18 @@ declare namespace Selectize {
         /**
          * "Selects" an item. Adds it to the list at the current caret position.
          */
-        addItem(value: T): void;
+        addItem(value: T, silent?: boolean): void;
 
         /**
          * Removes the selected item matching the provided value.
          */
-        removeItem(value: T): void;
+        removeItem(value: T, silent?: boolean): void;
 
         /**
          * Invokes the "create" method provided in the selectize options that should provide the data for the
          * new item, given the user input. Once this completes, it will be added to the item list.
          */
-        createItem(value: T): void;
+        createItem(value: T, triggerDropdown?: boolean, callback?: () => void): void;
```

Comment:
the callback is sometimes invoked with `data?: any`. see https://github.com/selectize/selectize.js/blob/32ee80a8ac488ac8ccda3da0252fcce86485bb0f/src/selectize.js#L1615

Reply:
Fixed :)